public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_log_analytics/azure_log_analytics_datasource.ts

```diff
@@ -170,6 +170,9 @@ export default class AzureLogAnalyticsDatasource extends DataSourceWithBackend<
   }
 
   async getWorkspaceDetails(workspaceId: string) {
+    if (!this.subscriptionId) {
```

Comment:
I'm not sure why this is necessary, it was for me to get things to load but I think is possibly unrelated to this change.